gcalcli/argparsers.py

```diff
@@ -103,9 +114,13 @@ def get_output_parser(parents=[]):
     output_parser.add_argument(
             '--width', '-w', default=10, dest='cal_width', type=validwidth,
             help='Set output width')
+    has_24_hours = locale_has_24_hours()
     output_parser.add_argument(
-            '--military', action='store_true', default=False,
+            '--military', action='store_true', default=has_24_hours,
             help='Use 24 hour display')
+    output_parser.add_argument(
+            '--no-military', action='store_false', default=has_24_hours,
```

Comment:
I think you're going to need a `dest` for this option if you want to integrate it with the current application code which is going to be looking for `military`.

Reply:
Fixed